src/styles/variables.css

```diff
@@ -14,4 +15,10 @@
 	--title-bar-color: var(--dojo-white);
 	--underlay-bg: var(--dojo-shade-gray);
 	--slide-panel-width: 256px;
+
+	--button-bg: var(--dojo-blue);
```

Comment:
I think we need to be more generic with these naming conventions. `--button` may be too specific when you consider that we will likely use these variables across multiple widgets.